qlib/kernel/task.rs

```diff
@@ -526,14 +566,9 @@ impl Task {
     }
 
     #[inline(always)]
-    pub fn TaskId() -> TaskId {
+    pub fn PrivateTaskID() -> u64 {
         let rsp = GetRsp();
-        return TaskId::New(rsp & DEFAULT_STACK_MAST);
-    }
-
-    #[inline(always)]
-    pub fn GetPtr(&self) -> &'static mut Task {
-        return unsafe { &mut *(self.taskId as *mut Task) };
+        return rsp & DEFAULT_STACK_MAST;
```

Comment:
typo "MAST", but perhaps fix in another PR.

Reply:
fixed